src/pages/index.js

```diff
@@ -1,27 +1,31 @@
 import Link from "next/link";
 
-export default function TransactionList({ initialData}) {
+export default function TransactionList({ transactions }) {
   // Sort transactions by date in descending order
-  const sortedTransactions = initialData.sort(
-    (a, b) => new Date(b.date) - new Date(a.date)
+  const sortedTransactions = transactions.sort(
+    (a, b) => new Date(b.date) - new Date(a.date),
   );
 
   return (
     <div>
       <h1>Transactions</h1>
-      <Link href="/transactions/TransactionForm">Add</Link>
+      <Link href="/transactions/create">Add</Link>
       <ul>
-        {sortedTransactions.map((transaction) => (
-          <li key={transaction.id}>
-            <Link href={`/transactions/${transaction.id}`}>
-              <h3>{transaction.category}</h3>
-              <div>{transaction.date}</div>
-              <div>
-                {transaction.amount.toFixed(2)} {transaction.currency}
-              </div>
-            </Link>
-          </li>
-        ))}
+        {sortedTransactions.map((transaction) => {
+          const displayAmount = `${transaction.transactionType === "Expense" && "-"}${transaction.amount.toFixed(2)}`;
+
+          return (
+            <li key={transaction.id}>
+              <Link href={`/transactions/${transaction.id}`}>
+                <h3>{transaction.category}</h3>
+                <div>{transaction.date}</div>
+                <div>
+                  {displayAmount} {transaction.currency}
+                </div>
```

Comment:
Do you really want to use ```<div>```for displaying the Date and Amount? Maybe a section including both infos as <p> or <hX> might be an alternative to avoid divs.

Reply:
this is just a placeholder and will be changed in the styling ticket